src/FlagProvider.tsx

```diff
@@ -14,50 +14,49 @@ const FlagProvider: React.FC<React.PropsWithChildren<IFlagProvider>> = ({
   config,
   children,
   unleashClient,
-  startClient = true,
+  startClient = typeof window !== undefined,
```

Comment:
Could we avoid a major release by implementing a new option and marking this one as deprecated, and then check both variables with an OR operator?